mcpjam-inspector/client/src/index.css

```
@@ -512,6 +512,95 @@
+   light theme). Keyframe endpoints are reconstructed to reproduce the sweep —
+   only the name, 200% size, 1.5s linear timing, and the resting `200% 50%`
+   position came across the wire. */
+.nds-shimmer-text {
+  --nds-shimmer-faded: #d4d3cf;
+  --nds-shimmer-solid: #7d7a75;
+  background-image: linear-gradient(
+    to right,
+    var(--nds-shimmer-faded) 0%,
+    var(--nds-shimmer-solid) 50%,
+    var(--nds-shimmer-faded) 100%
+  );
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)